src/member/routes.ts

```diff
@@ -0,0 +1,78 @@
+import { UUID } from '@graasp/sdk';
+
+import { DEFAULT_THUMBNAIL_SIZE } from '../config/constants.js';
+
+export const MEMBERS_ROUTE = `members`;
+
+export const buildGetCurrentMember = () => `${MEMBERS_ROUTE}/current`;
```

Comment:
Did you change this to uniformize to using only functions? And thus the ones below stayed as CAPITAL_LETTERS to avoid changing to many things?

Reply:
Yes I did not change the subscriptions routes as we are not using them currently.